src/pubtools/_marketplacesvm/cloud_providers/base.py

```diff
@@ -196,19 +198,19 @@ def _post_publish(
         """
         return push_item, publish_result
 
-    def _post_delete(self, push_item: T, **kwargs) -> T:
+    def _post_delete(self, push_item: T, delete_result: Any, **kwargs) -> Tuple[T, Any]:
         """
         Define the default method for post publishing actions.
 
         Args:
             push_item (VMIPushItem)
                 The push item to associate and publish a VM image into a product.
             delete_result (Any)
-                The resulting data from delete.
+                Tuple: The processed push item and the resulting data from delete.
```

Comment:
`Tuple` should be mentioned in the `Returns` section

the parameter `delete_result` is just Any

Reply:
Oops!

Fixing

Reply:
Fixed!